iidm-xml-converter/src/main/java/eu/itesla_project/iidm/xml/VscConverterStationXml.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2016, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.iidm.xml;
+
+import eu.itesla_project.iidm.network.VoltageLevel;
+import eu.itesla_project.iidm.network.VscConverterStation;
+import eu.itesla_project.iidm.network.VscConverterStationAdder;
+
+import javax.xml.stream.XMLStreamException;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ * @author Mathieu Bague <mathieu.bague at rte-france.com>
+ */
+class VscConverterStationXml extends ConnectableXml<VscConverterStation, VscConverterStationAdder, VoltageLevel> {
+
+    static final VscConverterStationXml INSTANCE = new VscConverterStationXml();
+
+    static final String ROOT_ELEMENT_NAME = "vscConverterStation";
+
+    @Override
+    protected String getRootElementName() {
+        return ROOT_ELEMENT_NAME;
+    }
+
+    @Override
+    protected boolean hasSubElements(VscConverterStation cs) {
+        return true;
+    }
+
+    @Override
+    protected void writeRootElementAttributes(VscConverterStation cs, VoltageLevel vl, XmlWriterContext context) throws XMLStreamException {
+        context.getWriter().writeAttribute("voltageRegulatorOn", Boolean.toString(cs.isVoltageRegulatorOn()));
+        XmlUtil.writeFloat("voltageSetPoint", cs.getVoltageSetPoint(), context.getWriter());
+        XmlUtil.writeFloat("reactivePowerSetPoint", cs.getReactivePowerSetPoint(), context.getWriter());
```

Comment:
voltageSetPoint and reactivePowerSetPoint are optionals (exclusives), so if one is NaN, it should not be written